services/core-web/src/components/mine/Permit/PermitAmendmentUploadedFilesList.tsx

```diff
@@ -28,6 +28,7 @@ export const PermitAmendmentUploadedFilesList = (props) => (
                 <p>Are you sure you want to remove this file?</p>,
                 <p>This cannot be undone.</p>,
               ]}
+              /* eslint react/jsx-key: 0 */
```

Comment:
Are these of type IPermitAmendmentMineDocument? Can we use that to assign a key?
If not, eslint-disable-next-line is a little easier to search

Reply:
:100:

Reply:
This has been adjusted to include keys now. I honestly thought we only needed to add a key in the parent element in React mappings, so I was really confused about the error I was getting, but yeah adding keys to <p> elements solves this issue, thanks!